### ISSUE TITLE
Driver unable to set FSGSBASE bit in kernel v5.4.50 and later versions

### ISSUE THREAD

**Issue by bvavala** [oscarlab/graphene-sgx-driver#31]
Starting from kernel v5.4.50 (see [changelog](https://cdn.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.4.50)) the FSGSBASE bit in CR4 is pinned -- roughly meaning that the kernel undoes any attempt to flip the bit.

This is visible in the [`native_write_cr4`](https://elixir.bootlin.com/linux/v5.4.50/source/arch/x86/kernel/cpu/common.c#L370) function, which is called by the `__write_cr4` function used in the driver.

As a result, although the driver loads successfully, FSGSBASE is not persistently enabled. In particular, it is disabled by the time `__enable_fsgsbase` returns.

**Comment by mkow** [comment]
That's true, but we don't plan to fix this, as this module is deprecated. The intended way to enable FSGSBASE is either through the patches for 5.4 or just by using Linux >= 5.9 (see https://graphene.readthedocs.io/en/latest/building.html#building-with-sgx-support).

Anyways, thanks for writing up a nice summary here, I'll pin the issue for others to make it more visible.
